source/filter.js

```diff
@@ -0,0 +1,37 @@
+"use strict";
+
+const map = {
+  "&": "&amp;",
+  "<": "&lt;",
+  ">": "&gt;",
+  '"': "&quot;",
+  "'": "&#39;",
+};
+
+
+/**
+ * Экранирует символы в html, оставляя только разрешенные теги
+ * @param {string} text - фильтруемый текст
+ * @param {Array<string>} tags - список разрешенных тегов
```

Comment:
```suggestion
 * @param {string[]} [tags=[]] - список разрешенных тегов
```
`Array` писать правильно - мне нравиться больше такая запись + указание дефолтного значения у `tags`